keras_retinanet/bin/train.py

```diff
@@ -406,6 +428,12 @@ def csv_list(string):
     parser.add_argument('--image-min-side', help='Rescale the image so the smallest side is min_side.', type=int, default=800)
     parser.add_argument('--image-max-side', help='Rescale the image if the largest side is larger than max_side.', type=int, default=1333)
 
+    parser.add_argument('--tensorboxes', dest='tensorboxes',  help='Number of images to store on tensorboard', type=int, default=0)
```

Comment:
The `dest` is already `tensorboxes` by default, why assign it? (same for `optimizer`, `lr`, `clip_norm`).